app/views/evidence_hub/_checkboxes.html.erb

```diff
@@ -0,0 +1,10 @@
+<% form_object.class.human_attribute_name(field).each do |option| %>
+  <%= f.label field, {class: "checkbox-label"} do %>
+    <%= f.check_box field, 
+      { class: "checkbox", name: "#{form_object.class_name}[#{field}][]", checked: form_object.chosen_options?(option, field) }, 
+      option, 
+      nil 
+    %>
+    <%= option %>
+  <% end %>
+<% end %>
```

Comment:
Even if the label is wrapped inside a block, the label for and the input name and id are the same, which generates problems with label and input:

```
<label class="checkbox-label" for="evidence_summary_search_form_client_groups">
    <input class="checkbox" name="evidence_summary_search_form[client_groups][]" type="checkbox" value="Children (3 - 11)" id="evidence_summary_search_form_client_groups">
    Children (3 - 11)
</label>
<label class="checkbox-label" for="evidence_summary_search_form_client_groups">
    <input class="checkbox" name="evidence_summary_search_form[client_groups][]" type="checkbox" value="Young people (12 - 16)" id="evidence_summary_search_form_client_groups">
    Young people (12 - 16)
</label>
```

So the label is not working. When you click the last element, the first element is selected. A suggestion would be using an unique identifier (by the index number of the array OR by the parametrize value of the option eg young_people)